extensions/cornerstone/src/init.tsx

```diff
@@ -104,6 +105,12 @@ export default async function init({
     clearOnModeExit: true,
   });
 
+  // Stores the entire ViewportGridService getState when toggling to one up
```

Comment:
Nice to have this commented here - makes it a lot easier to understand.

Reply:
Thanks.